sbnobj/Common/Reco/ScatterClosestApproach.h

```diff
@@ -0,0 +1,29 @@
+// Class for holding the result of calculating the Distacnce of Closest approach for a track
+
+#ifndef sbncode_scatterclosestapproach_H
+#define sbncode_scatterclosestapproach_H
+
+namespace sbn {
+class ScatterClosestApproach {
+  public:
+  ScatterClosestApproach()
+      : mMean(-5.f)
+      , mStdDev(-5.f)
+      , mMax(-5.f)
+  {
+  }
+
+  ScatterClosestApproach(const float mean, const float stdDev, const float max)
+      : mMean(mean)
+      , mStdDev(stdDev)
+      , mMax(max)
+  {
+  }
+
+  float mMean;   // Average ClosestApproach [cm]
```

Comment:
We use `f` for fields elsewhere. But I also think for a "struct-like" data product like this where the fields are directly accessible I just wouldn't prefix at all.

Reply:
sorry, can we lowercase the initial letters too? (I assume `max` doesn't cause trouble by clashing with a function?)

Reply:
Done